### ISSUE TITLE
Incorrect role is defined for menuitems "User", "Workspace" and "Remote[Azure ML: v-...]" as checkboxes.: A11y_Azure Machine Learning Extension for VS Code_Settings_AI4D

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode-tools-for-ai#2376]
"[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier."
### GitHub Tags
#A11yTCS; #A11ySev2; #WCAG4.1.2; #BM_AzureMachineLearningExtensionforVSCode_Win32_May2024; #DesktopApp; #Azure Machine Learning Extension for VS Code; #A11yMAS; #Name Role Value; #AILimited; #Win11; #AI4D; 

### Environment Details:
Application Name: Azure Machine Learning Extension for VS Code
Visual Studio Code: Version 1.89.1(User setup)

### Repro Steps:
1. Install VS Code
2. Install the Azure Machine Learning extension and sign in with appropriate Credentials.
3. Open the Command Palette via the keyboard command Ctrl+Shift+P (or Command+Shift+P for MacOS)
4. Type "Open User Settings" and select the option from the dropdown list called "Preferences: Open User Settings") and press enter
5. In the search box, type "Azure Machine Learning"
6. Tab until the list of settings is active
7. Use the up/down arrows to navigate between settings
8. Tab till "User" menuitem and Verify whether the correct role is defined for "User", "Workspace" and "Remote[Azure ML: v-...]" or not.
### Actual Result:
Incorrect role is defined for menuitems "User", "Workspace" and "Remote[Azure ML: v-...]" as checkboxes

### Expected Result:
Correct role should be defined for "User", "Workspace" and "Remote[Azure ML: v-...]" as menuitems.

### User Impact:
Screen reader users will face difficulty in accessing the control if the correct control type is not defined for the controls.

### Attachment:
![A11y_Azure Machine Learning Extension for VS Code_Settings_AI4D](https://github.com/microsoft/vscode-tools-for-ai/assets/98530069/51dd78be-0707-4b28-8286-84a1c1ef22d0)

**Comment by Yash14j** [comment]
GitHub Tags:#Rev:yaja;

**Comment by tbombach** [comment]
Thank you so much for filing this @kupatkar99!

This is UI provided by VS Code that displays all settings (both VS Code settings and any extension settings). It allows the user to select which settings to edit (for example, their user settings or settings just for the local folder or workspace). The role is set in VS Code. The guidelines suggest that I should re-assign this item back to you so you can file the bug for that product (please let me know if there is a different process I should follow). The VS Code repository is located here: https://github.com/microsoft/vscode 

I found this discussion of the role for [User, Workspace, etc] items in the Settings Scope Switcher: https://github.com/microsoft/vscode/issues/97068 I think this will be useful context for VS Code team for this item. I will be happy to follow up with VS Code after the issue is filed to figure out the right fix for this.

**Comment by tbombach** [comment]
I'm following up with VS Code on the external bug for a fix in that product.

For the Azure ML Extension, we've made the default settings so that a user does not need to access this UI for any core scenarios. The user would only need to access settings for optional, preview features (for example, testing an experimental version of schemas). So the settings UI is not a part of core scenarios for this product.

@kupatkar99 Could this be lowered to a severity 3 issue since it no longer impacts a core scenario?

**Comment by kupatkar99** [comment]
As Per the discussion with Indu, this bug need to be fixed and we can't lower the severity as it's impacting the users in accessing the Control if incorrect role is defined for the control.

**Comment by tbombach** [comment]
There is now a fix for the upstream issue: https://github.com/microsoft/vscode/issues/214227

So we will not have to change the severity of this item. Thank you @kupatkar99!

**Comment by kupatkar99** [comment]
verified the issue in below environment:
Visual Studio Code Insider
Version: 1.92.0-insider (user setup)
Issue is fixed. Hence Closing the bug please find added attachment for reference.
![2376_Fixed](https://github.com/user-attachments/assets/dc4e2fce-660e-4498-82be-68ff389c61b8)
